src/Filters/molecules/FilterInputs.tsx

```diff
@@ -1,15 +1,50 @@
 import React, { useState, useCallback } from 'react';
-import styled from 'styled-components';
+import styled, { css, keyframes } from 'styled-components';
 import BasicSearchInput, { IBasicSearchInput } from '../../Misc/atoms/BasicSearchInput';
 import FilterDropdown, { IFilterDropdown } from '../../Filters/molecules/FilterDropdown';
 import FilterButton from '../../Filters/atoms/FilterButton';
+import Icon, { IconWrapper } from '../../Icons/Icon';
+import { resetButtonStyles } from '../../common/index';
+
+const fadeInAnimation = keyframes`
+  0% {
+    width: 50%;
+  }
+  100% {
+    width: 100%;
+  }
+`;
```

Comment:
Animation looks great. Could we add a little opacity to the transition too?

I think for the whole thing might be too much so perhaps at 0% `opacity: 0;` is good and then maybe hit `opacity: 1` at around 50% could be good.